sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.controller.service.ts

```diff
@@ -40,4 +56,70 @@ export class ScholasticStandingControllerService {
       ...transformToActiveApplicationDataAPIOutDTO(application, offering),
     };
   }
+
+  /**
+   * Verify if all text file validations were performed with success and throw
+   * a BadRequestException in case of some failure.
+   * @param textValidations validations to be verified.
+   */
+  assertTextValidationsAreValid(
```

Comment:
Thanks for moving this from the controller to the controller service 😉

Reply:
Thanks for suggesting the change.

Reply:
Good one to move